src/css/components/_notification.scss

```diff
@@ -61,10 +73,10 @@
     }
     &__message {
         color: var(--text-color, #000);
-        padding: 0 2rem;
+        padding: var(--c-notify__message--padding);
         text-align: center;
         span {
-            font-size: 16px;
+            @include text--paragraph;
```

Comment:
This line is in response to [the feedback](https://github.com/telosnetwork/telos-wallet/pull/373#discussion_r1212039168) from @karynemayer in the original PR.
Thank you, Karyne